src/content/docs/release-notes/mobile-release-notes/ios-release-notes/ios-agent-742.mdx

```diff
@@ -0,0 +1,23 @@
+---
+subject: iOS agent
+releaseDate: '2023-02-23'
+version: 7.4.2
+downloadLink: 'https://download.newrelic.com/ios_agent/NewRelic_XCFramework_Agent_7.4.2.zip'
+---
+
+## Fixed in this release
+
+* Fixed an issue with the agent that could cause Apple App Store rejection.
+
+### Other notes
+* Apple no longer supports bitcode-based submissions to the App Store. The New Relic iOS agent should not be used with bitcode-based apps. The included dSYM upload script will not work with bitcode-based apps.
+
+* Deprecation notice: The 7.4.0 version introduced a new Swift symbol upload script. See [installation instructions](/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation#configure-using-swift-package-manager) for details on how to update to the new script. In a future version of the iOS agent, the Python-based symbol upload scripts will be removed.
+* This artifact was built using Xcode 14.2 with a minimum deployment target of iOS version 9.0. Compatible with CocoaPods 1.10 or higher and Swift Package Manager. The agent should be used with the latest Xcode version (Xcode 14.2 at the time of release).
+* For Mac Catalyst, the zip file installation may integrate more easily than using CocoaPods.
+
+### Support statement
+
+* As of this release, the iOS agent will consolidate previously separate XCFramework and tvOS agents into a singular iOS agent. 
```

Comment:
```suggestion
```
@cdillard-NewRelic Should we remove this line or update it to say "As of iOS agent 7.4.1, the iOS agent has consolidated previously separate XCFramework and tvOS agents into a singular iOS agent."

Reply:
I think its okay as is for now. We can update next time to say 7.4.2 consolidated into singular iOS agent. (Even though technically the truth is that xcframework replaced platform specific agents years ago. the xcframework agent has simply been been renamed to iOS agent)